corehq/apps/reports/templates/reports/filters/drilldown_options.html

```diff
@@ -5,7 +5,12 @@
 {% if is_empty %}
 <div class="alert alert-info">{{ empty_text }}</div>
 {% else %}
-<div id="{{ css_id }}" class="well well-sm">
+<div id="{{ css_id }}" class="well well-sm report-filter-drilldown-options"
+     data-drilldown-map='{{ option_map|JSON }}'
```

Comment:
FYI, there's a `{% html_attr option_map %}` that serializes as JSON and also does HTML attribute escaping. I think JSON escape single quotes so you're fine here and it's `data-foo="{{ foo|JSON }}"` that's the risk.

Reply:
Yeah, it was odd at first that double quotes didn't work, but I figured that was the reason